pydantic/_internal/_core_metadata.py

```diff
@@ -4,32 +4,32 @@
 from typing import Any
 
 import typing_extensions
-from pydantic_core import CoreSchema
 
 if typing.TYPE_CHECKING:
-    from ..json_schema import JsonSchemaValue
     from ._schema_generation_shared import (
         CoreSchemaOrField as CoreSchemaOrField,
     )
     from ._schema_generation_shared import (
         GetJsonSchemaFunction,
-        GetJsonSchemaHandler,
     )
 
 
 class CoreMetadata(typing_extensions.TypedDict, total=False):
+    """A TypedDict for holding the metadata dict of the schema
+
+    Attributes:
+        pydantic_js_functions: List of Json schema functions.
```

Comment:
```suggestion
        pydantic_js_functions: List of JSON schema functions.
```

Reply:
@hramezani I see you marked this resolved but didn't change it; I also support this change